community/cloud-foundation/templates/gcs_bucket/examples/gcs_bucket_iam_bindings.yaml

```diff
@@ -1,17 +1,20 @@
-# Example on how to use the cloud storage bucket template.
+# Example usage of the Google Cloud Storage bucket template.
```

Comment:
`Google Cloud Storage bucket` and `Google Cloud Storage Bucket` are being used in this template; can we please normalize to `Google Cloud Storage Bucket`. Please check all files in this template and update where required. thx.

Reply:
@fawix , "bucket" is spelled with small "b" because it is a "generic" (not Google-branded) word (unlike "Cloud Storage", which is Google-branded, and therefore is initial-capitalized). "Bucket" is initial-capitalized in titles, where all words are initial-capitalized. This is the logic that we tried to adhere to in all templates... Hope you agree with it.

Reply:
Oh ok, I see; Makes sense I'll resolve this conversation thanks!